teams/goals.toml

```diff
@@ -0,0 +1,35 @@
+name = "goals"
+subteam-of = "launching-pad"
+
+[people]
+leads = ["nikomatsakis"]
+members = [
+    "nikomatsakis",
+    "joshtriplett",
+    "lqd",
+    "nandsh",
+]
+alumni = [
+]
+
+[[github]]
+team-name = "goals"
+orgs = ["rust-lang"]
+
+[permissions]
+bors.rust.review = true
+
+[rfcbot]
+label = "T-goals"
+name = "Goals"
+ping = "rust-lang/goals"
+
+[website]
```

Comment:
Although it is good to include the website, just want to let you know that it won't show up on the website. It currently isn't rendering "teams" in the launching pad. I might look into fixing that.

Reply:
OK. That's fine, I'm just setting up the metadata.